components/Gallery.tsx

```diff
@@ -17,7 +17,7 @@ function Faqs({}: Props) {
         Gallery
       </h3>
 
-      <div>
+      <div className="w-full flex space-x-5 overflow-x-scroll p-10 snap-x snap-mandatory">
```

Comment:
oh are you trying to emulate a grid here?  there's already pretty good grid support in Tailwind without needing to roll your own

Reply:
This is for a side scrolling snap feature. Gives it a nice little transition with this way.

Reply:
ah right so they are carousels

Reply:
Yeah basically